src/app/game/game.component.html

```diff
@@ -1,56 +1,75 @@
 <div class="row" style="padding: 1rem;">
-    <div class="col-lg-6">    
+    <div class="col-lg-6"> 
+
       <div class="bs-component">
         <div class="card" style="text-align: center;">
           <div class="card-body">
               Your ships
           </div>
         </div>
         <app-ship-map></app-ship-map>
-
-    </div>
+      </div>
 
     </div>
     <div class="col-lg-6">
+
       <div class="bs-component">
         <div class="card" style="text-align: center;">
           <div class="card-body">
               Take a shot
           </div>
         </div>
-            <app-shoot-map></app-shoot-map>
-        </div>
+        <app-shoot-map></app-shoot-map>
+      </div>
+
     </div>
 </div>
 
 <div class="row" style="padding: 1rem;">
+  
   <div class="col-lg-4">
-          <div class="card" style="text-align: center; padding-top: 1rem;">
-            <h5 class="card-title">Player info</h5>
-            <div class="card-body">
-              <p *ngIf="player" >
-                Player {{player.id}}: {{player.name}}
-              </p>
-            </div>
-          </div>
-        </div>
+    <div class="card" style="text-align: center; padding-top: 1rem;">
+      <h5 class="card-title">Player info</h5>
+      <div class="card-body">
+        <p *ngIf="player" >
+          Player {{player.id}}: {{player.name}}
```

Comment:
Player.id będzie usunięty, żeby był zgodny z backendem, gdzie id nie ma. Jak możez to też usuń wszelkie zalezności od Player.id

Reply:
W kolejnej iteracji można całkowicie przbudować komunikację dla gracza - na osobnej PRce.